gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -74,17 +74,17 @@ <h3 class="mdc-layout-grid__cell--span-3">
           <div class="mdc-layout-grid__cell--span-9">
             <dl>
               <dt>Package Name</dt>
-              <dd>{{ affected['package']['name'] or 'Unknown' }}</dd>
+              <dd>{{ affected['package']['name'] or vulnerability.repo | strip_scheme }}</dd>
```

Comment:
Actually, thinking more on this, this doesn't seem very correct since this isn't actually a "Package Name", and it's also a bit repetitive.  IMO we should just strip this heading completely.

Reply:
Removed in https://github.com/google/osv.dev/pull/1483/commits/f6d276ead199d90463abb8edad478f2b59b82286